Jenkinsfile

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+// The declarative agent is defined in yaml.  It was previously possible to
+// define containerTemplate but that has been deprecated in favor of the yaml
+// format
+// Reference: https://github.com/jenkinsci/kubernetes-plugin
+pipeline {
+  agent {
+    kubernetes {
+      label 'k8s-infra'
+      defaultContainer 'jnlp'
+      yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    jenkins: build-node
+spec:
+  containers:
+  - name: k8s-node
+    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:1.0.0
```

Comment:
This is an older version of the builder. We need the latest tag. Victor know which one